sdk/servicebus/Azure.Messaging.ServiceBus/src/Receiver/ServiceBusReceiver.cs

```diff
@@ -253,28 +252,6 @@ public class ServiceBusReceiver : IAsyncDisposable
             return null;
         }
 
-        private async Task ApplyPlugins(IList<ServiceBusReceivedMessage> messages)
-        {
-            foreach (ServiceBusPlugin plugin in _plugins)
-            {
-                string pluginType = plugin.GetType().Name;
-                foreach (ServiceBusReceivedMessage message in messages)
```

Comment:
Note that with the current implementation, we execute all plugins for one message at a time as opposed to the previous where 1 plugin would run for all messages before going to the next plugin.